<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlStrip</class>
 <widget class="QWidget" name="ControlStrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="playPauseButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="elapsedLabel">
     <property name="text">
      <string notr="true">00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="totalLabel">
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fullScreenButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="view-fullscreen"/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
